src/resources/ApiKeys/ApiKeysInterfaces.ts

```diff
@@ -84,6 +84,14 @@ export interface ApiKeyModel extends GranularResource {
      * Additional configuration to be included in an API key. [to be revised]
      */
     additionalConfiguration?: AdditionalConfigurationModel;
+    /**
+     * The expiration date of the api key
+     */
+    expirationDate?: number;
+    /**
+     * Represents the status of the api key 
+     */
+    status?: ApiKeyStatus
 }
```

Comment:
While at it, I think we should make the following corrections to the ApiKeyModel:
- The `id` should not be optional, an api key always has an id
- The `apiKeyTemplateId` field is missing

Reply:
Perfect on it